tremor-script/src/srs.rs

```diff
@@ -17,9 +17,200 @@ use crate::{
     errors::{Error, Result},
     prelude::*,
 };
+use halfbrown::HashMap;
 use std::{fmt::Debug, mem, pin::Pin, sync::Arc};
 
-///! Thisn file includes our self referential structs
+/// A fully resolved deployable artefact
+#[derive(Clone, Debug, PartialEq)]
+pub enum AtomOfDeployment {
```

Comment:
`<'static>` where will violate the SRS guarantees as the enum doesn't include the corresponding vectors.

Reply:
Yes - i may have forgotten to check in the updates for connector/flow here. I'll redo. Well spotted

Reply:
Done